docs/release.md

```diff
@@ -137,6 +137,16 @@ The last step of the release process creates artifacts for the new version of th
 
 9. Update the release notes with the CHANGELOG.md updates.
 
+## Post-release steps
+
+After the release is complete, the release manager should do the following steps:
+
+1. Create an issue or update existing issues for each problem encountered throughout the release in
+the appropriate repositories and label them with the `release:retro` label. The release manager
+should share the list of issues that affected the release with the Collector leads.
+2. Update the [release schedule](#release-schedule) section of this document to remove the first
+   line and add themselves to the bottom of the list.
```

Comment:
```suggestion
2. Update the [release schedule](#release-schedule) section of this document to remove the completed
releases and add new schedules to the bottom of the list.
```

just in case the previous releases were left out